PCLConfig.cmake.in

```diff
@@ -124,7 +124,7 @@ macro(find_eigen)
   elseif(NOT EIGEN_ROOT)
     get_filename_component(EIGEN_ROOT "@EIGEN_INCLUDE_DIRS@" ABSOLUTE)
   endif()
-  find_package(Eigen)
+  find_package(Eigen 3.1)
```

Comment:
Shouldn't this be `REQUIRED` as well?

Reply:
Nop, this is handled in `find_external_library`

Reply:
Minor comment: I feel you're marking these comments as resolved  a little early :/ In this case for instance, you replied to my question, but this doesn't necessarily mean I understood your reply and it does not entail that just because you replied, my comment/remark is now resolved. In general, unless it's super obvious that some point as been addressed and understood by the person who launched the discussion it's not a good idea to mark the comment/line as resolved, this is particularly true if you're not the "comment/discussion starter".

I'm mentioning this because I noticed I gained the habit of rechecking "all resolved conversations" in your PRs.